Added Wand Details for Professor McGonagall

SunSummoner opened the pull request:
Got the data from wiki fandom website/

SunSummoner committed:
Added McGonagall's wand details

I got the details from Wiki Fandom website.

SunSummoner committed:
Merge pull request #1 from SunSummoner/SunSummoner-patch-1

Added McGonagall's wand details

KostaSav reviewed (changes requested):
Hi @SunSummoner ,

Thank you for your contribution, but I think there might be something wrong with your commit - I see you put the wand details in the ancestry and eyeColour fields. 

Can you please check and edit your pull request?

SunSummoner commented:
> Hi @SunSummoner ,
> 
> Thank you for your contribution, but I think there might be something wrong with your commit - I see you put the wand details in the ancestry and eyeColour fields. 
> 
> Can you please check and edit your pull request?

Yeah, I'll correct that mistake. Thanks for pointing that out.

SunSummoner committed:
Corrected the wand details

I removed the wand details from eye colour and ancestry. Sorry for the trouble.

SunSummoner committed:
Merge pull request #2 from SunSummoner/SunSummoner-patch-2

Corrected the wand details
